src/MissionManager/SurveyComplexItem.cc

```diff
@@ -196,6 +201,7 @@ bool SurveyComplexItem::_loadV4(const QJsonObject& complexObject, int sequenceNu
 
     _gridAngleFact.setRawValue              (complexObject[_jsonGridAngleKey].toDouble());
     _flyAlternateTransectsFact.setRawValue  (complexObject[_jsonFlyAlternateTransectsKey].toBool(false));
+    _splitConcavePolygonsFact.setRawValue   (complexObject[_jsonSplitConcavePolygonsKey].toBool(true));
```

Comment:
I believe this will assert if the key is missing (which it will be in older files). How to handle this will depend on how you handle the version bump from 4 to 5. Following how I handled this In other places with a similar version bump:
* Save sets version to 5
* Load: Make the key required if version=5
* Load: Do a contains check before accessing on load
* Load: If not there, set to false

Reply:
>I hate this f'ing git review thing

Tell me about it. I get completely lost once you get to more than a couple of comments.

>If you look at FixedWingLandingComplexItem you can see an example it a complex item which has gone through many version bumps and how it was handled on load.

Will do